examples/07.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import elabapi_python
+
+#########################
+#         CONFIG        #
+#########################
+API_HOST_URL = 'https://elab.local:3148/api/v2'
+# replace with your api key
+API_KEY = 'apiKey4Test'
+#########################
+#      END CONFIG       #
+#########################
+
+# Configure the api client
+configuration = elabapi_python.Configuration()
+configuration.api_key['api_key'] = API_KEY
+configuration.api_key_prefix['api_key'] = 'Authorization'
+configuration.host = API_HOST_URL
+configuration.debug = False
+configuration.verify_ssl = False
+
+# create an instance of the API class
+api_client = elabapi_python.ApiClient(configuration)
+# fix issue with Authorization header not being properly set by the generated lib
+api_client.set_default_header(header_name='Authorization', header_value=API_KEY)
+
+# Create instance of ExperimentTemplateApi
+templateApi = elabapi_python.ExperimentsTemplatesApi(api_client)
+
+# Template that we want to create, exemple with extra fields
```

Comment:
```suggestion
# Template that we want to create, example with extra fields
```

Reply:
Typical french issue, will modify that thank you !